ccp/docker-compose.yml

```diff
@@ -104,7 +106,17 @@ services:
       - "forward_proxy"
     volumes:
       - /etc/bridgehead/trusted-ca-certs:/conf/trusted-ca-certs:ro
-
+  monitoring:
+    image: "samply/bridgehead-monitoring:main"
+    container_name: bridgehead-monitoring
+    environment: 
+      HOST: ${HOST}
+      SITE_NAME: ${SITE_NAME}
+      PROJECT: ${PROJECT}
+      BEAM_URL: http://beam-proxy:8081
+      PROXY_ID: monitoring.${PROXY_ID}
+      MONITORING_TARGET: ${MONITORING_TARGET}
```

Comment:
Please derive the `MONITORING_TARGET` from the project (you can use `$PROJECT/vars` file) rather than a site-specific config. For the simple reason that it's not site specific.